src/briefcase/platforms/windows/__init__.py

```diff
@@ -180,6 +180,10 @@ def run_app(
             )
 
 
+class WindowsDevCommand(DevCommand):
```

Comment:
Since we're not anticipating any difference between Windows app an Windows VisualStudio development modes, having this intermediate class is probably overkill. The macOS analog shows how you can avoid this intermediate class.